packages/contracts/contracts/test/Colosseum.t.sol

```diff
@@ -77,16 +78,17 @@ contract Colosseum_Test is Colosseum_Initializer {
 
         for (uint256 i = 0; i < segLen; i++) {
             uint256 n = _segStart + i * (_segSize / (segLen - 1));
-            arr[i] = _sender == asserter ? _invalidOutputs[n] : _actualOutputs[n];
+            arr[i] = _getOutputRoot(_sender, n);
         }
 
         return arr;
     }
 
-    function _detectFault(
-        Types.Challenge memory _challenge,
-        address _sender
-    ) private view returns (uint256) {
+    function _detectFault(Types.Challenge memory _challenge, address _sender)
+        private
+        view
+        returns (uint256)
+    {
```

Comment:
Could you restore to the original code? The original code is the right code which conforms to formatting rule.